### ISSUE TITLE
Parallel cross validation is unsafe to use

### ISSUE THREAD

**Issue by lkuchenb** [shogun-toolbox/shogun#3727]
* The cloning of some machines is actually broken (LibSVM, MKL, see #3725)
 * Some machines use 3rd party libs that aren't threads safe (see #3726)

I suggest to pull parallel cross val out until those are resolved.

**Comment by karlnapf** [comment]
Yes, and next time we need to be more careful with merging things like this
@lkuchenb you want to send a revert patch?

**Comment by lkuchenb** [comment]
Parallel xval was disabled again with #3742
